yast2_firewall_broadcast.json

```diff
@@ -0,0 +1,15 @@
+{
+  "tags": [
+    "yast2_firewall_broadcast"

```

Comment:
This is okay as I can see. Please see example: http://e13.suse.de/tests/1820#step/yast2_firewall/13